pygmt/src/solar.py

```diff
@@ -0,0 +1,97 @@
+"""
+solar - Plot day-night terminators.
+"""
+
+import datetime
+
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    J="projection",
+    G="fill",
+    B="frame",
+    W="pen",
+    U="timestamp",
+    V="verbose",
+    X="xshift",
+    Y="yshift",
```

Comment:
Sort alphabetically.

```suggestion
    B="frame",
    G="fill",
    J="projection",
    R="region",
    U="timestamp",
    V="verbose",
    W="pen",
    X="xshift",
    Y="yshift",
```